src/epython/poke/eprequests.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+"""
+Description:
+    This module houses the Poke class, and the wrapped requests methods.
+
+Author:
+    Ray Gomez
+
+Date:
+    3/16/21
+"""
+
+import requests
+
+from environment import EPYTHON_REQUEST_ID
+from handlers import basic_retry_handler
+
+POKE_HEADERS = {
+    "Content-Type": "application/json",
+    "Accept": "application/json",
+    "X-Request-ID": EPYTHON_REQUEST_ID
+}
+
+# These are the most common requests exceptions that will trigger a retry
+# (this is explicit to show what is actually being retried on)
+COMMON_REQUEST_EXCEPTIONS = (requests.exceptions.RequestException,
+                             requests.exceptions.HTTPError,
+                             requests.exceptions.ConnectionError,
+                             requests.exceptions.Timeout,
+                             requests.exceptions.ConnectTimeout,
+                             requests.exceptions.ReadTimeout)
+
+
+def get(url, params=None, data=None, auth=None, headers=None, timeout=None,
```

Comment:
Is there a default timeout in the lower-level requests code we're calling?

Reply:
Nope, that got stripped with my latest round of clean up, this is a great call out though. I'll add these to the environment variables as well.

Reply:
Implemented!